src/sx.ts

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types'
```

Comment:
Not a big deal, just for future reference, you can use `git mv` to rename a file if the heuristic isn't strong enough to detect it.

Reply:
Good to know. Thanks!